test/sensor_simulator/sensor_simulator.cpp

```diff
@@ -12,8 +12,8 @@ _rng(ekf),
 _vio(ekf),
 _airspeed(ekf)
 {
-	setSensorDataToDefault();
 	setSensorRateToDefault();
+	setSensorDataToDefault();
```

Comment:
@kamilritz This was the issue making macOS unit test fail. In fact, the unit test failed because dt was set to 0 but on Linux, the value was corrected to 0.02 in one of the tests above (-> removing the tests above the flow one triggered the issue). On macOS, it seems that the variables are reset between each unit test (which is good).